sage-ios/SAGE/SAGE/SchoolDetailViewController.swift

```diff
@@ -34,6 +36,31 @@ class SchoolDetailViewController: UITableViewController {
         super.init(coder: aDecoder)
     }
     
+    func editedProfile(notification: NSNotification) {
+        let newUser = notification.object!.copy() as! User
+        if let school = self.school {
+            var reload = false
+            
+            if newUser.id == school.director?.id {
+                reload = true
+                school.director = newUser
+            }
+            if var students = school.students {
+                for var index = 0; index < students.count; ++index {
+                    let student = students[index]
+                    if student.id == newUser.id {
+                        reload = true
+                        self.school!.students![index] = newUser
+                    }
+                }
+            }
+            
+            if reload {
```

Comment:
dont need this reload thing instead you can reload a specific cell at an indexPath, which you will be able to get by the index of the user in the studentsArray